systemtest/src/main/java/io/strimzi/systemtest/security/SystemTestCertManager.java

```diff
@@ -100,12 +100,29 @@ public static SystemTestCertAndKey generateIntermediateCaCertAndKey(SystemTestCe
                 .withSubjectDn(STRIMZI_INTERMEDIATE_CA)
                 .build();
     }
+
+    public static SystemTestCertAndKey generateIntermediateCaCertAndKey(final SystemTestCertAndKey rootCert,
+                                                                        final ASN1Encodable[] sanDnsNames) {
+        return intermediateCaCertBuilder(rootCert)
+                .withSubjectDn(STRIMZI_INTERMEDIATE_CA)
+                .withSanDnsNames(sanDnsNames)
+                .build();
+    }
```

Comment:
Why is this needed? Is this even used anywhere? The SANs in the CA should not matter.

Reply:
We use it to generate:
1. Root CA
2. Intermediate CA
3. Strimzi CA

The intermediate CA is used for chain cert e.g.:
```java
SystemTestCertAndKey root1 = generateRootCaCertAndKey();
SystemTestCertAndKey intermediate1 = generateIntermediateCaCertAndKey(root1);
SystemTestCertAndKey strimzi1 = generateEndEntityCertAndKey(intermediate1);
ROOT_CA_CERT_AND_KEY_1 = exportToPemFiles(root1);
STRIMZI_CERT_AND_KEY_1 = exportToPemFiles(strimzi1);
CHAIN_CERT_AND_KEY_1 = exportToPemFiles(strimzi1, intermediate1, root1);
```

Reply:
The point is not whether you use Intermediate CA. The point is that you should not care about SANs for Intermediate CA (neither for Root CA). And you do not seem to use anywhere this particular method with the second parameter.

Reply:
Well, we use it in the test cases where I create whole chain cert (root, inter, strimzi) but I think as you mention the SANs should not matter so I will remove it.

Reply:
> The point is not whether you use Intermediate CA. The point is that you should not care about SANs for Intermediate CA (neither for Root CA). And you do not seem to use anywhere this particular method with the second parameter.

Yes, that's true and thanks for checking that :) <3